### PR TITLE
DOCSP-28834 release notes 1.3

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Adds release notes for mongosync 1.3.0/

## Staging

[RN](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-28834-release-notes-1.3/release-notes/1.3/)

## Jira

[DOCSP-28834](https://jira.mongodb.org/browse/DOCSP-28834)

## Build

[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64381b74b49106a498b489b7)

### CODE REVIEW

**Review by mdb-ashley — APPROVED** [review]
LGTM! Thanks, Kenneth.

File: source/release-notes/1.3.txt [hunk 1]
```diff
@@ -9,26 +9,41 @@ Release Notes for ``mongosync`` 1.3
...
+~~~~~~~~~~~~~~~
+
+Starting in 1.3.0, {+c2c-product-name+} includes the ``mongosyncID`` and
+``coordinatorID`` in the :ref:`c2c-api-progress` response documents.  This
+helps determine whether the ``mongosync`` instance is serving as a coordinator.
+
+Bug Fixes
+----------
+
+* Sync cluster collections on replica sets fail to copy all documents to the
```
Comment (rohshar) [inline source/release-notes/1.3.txt:48]:
```suggestion
* Syncing clustered collections on replica sets fails to copy all documents to the
```

File: source/release-notes/1.3.txt [hunk 2]
```diff
@@ -9,26 +9,41 @@ Release Notes for ``mongosync`` 1.3
...
+  destination cluster (:issue:`REP-1479`)
```
Comment (rohshar) [inline source/release-notes/1.3.txt:49]: Isn't the issue (REP-1479) internal?

Comment (kennethdyer) [reply source/release-notes/1.3.txt:49]: @rohshar, Not sure off hand.  I can cut the link if you don't want it there.

Comment (rohshar) [reply source/release-notes/1.3.txt:49]: Yeah I would just rephrase the whole statement to say:

"Fixes a bug that could cause ``mongosync`` to not copy some documents to the destination cluster for clustered collections on replica sets."

Comment (kennethdyer) [reply source/release-notes/1.3.txt]: Fixed

**Review by rohshar — APPROVED** [review]

**Pull request merged**